### PR TITLE
Scrolling

### PR DESCRIPTION
**Pull request by KCraw**
Fixed a bunch of shit and got scrolling recommendations working...

Didn't touch recommendation component itself really, natch.

### CODE REVIEW

File: components/navigation/index.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import React, {
```
Comment (danmconrad) [inline components/navigation/index.js]: At first I didn't like this change- having everything moved from the app. But then I started thinking of this "navigation" file as a "router" instead.

File: ios/gustave/AppDelegate.m [hunk 2]
```diff
@@ -31,7 +31,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
    * on the same Wi-Fi network.
    */
 
-  jsCodeLocation = [NSURL URLWithString:@"http://Conrad-Bodoni-2.local:8081/index.ios.bundle?platform=ios&dev=true"];
+  jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle?platform=ios&dev=true"];
```
Comment (danmconrad) [inline ios/gustave/AppDelegate.m]: best change so far.

**Comment by danmconrad** [comment]
I'm guessing the commented code is stuff you're about to implement, so I didn't bother commenting. In terms of CQ, there were very few things I'd mention so I'll just change them later. Architecturally I don't have enough context on this until we sit down together next. I say let's merge and continue forward. That way I can start working on the recommendation-scene. 

One quick bit- I'll have to rename some stuff, such as `navigation-bar` to something that makes more sense when we add in the top nav on the saved scene. I wonder if nesting components makes sense.

File: components/navigation/index.js (hunk 1 above)
Comment (KCraw) [inline components/navigation/index.js]: That's kinda how I was thinking about it. Almost named it 'router'.

**Comment by KCraw** [comment]
There is so much reorg I want to do when we move past the prototype. This shit was just driving me nuts to reason about so I tried to simplify. We get past the prototype, we can spend a lot of time architecting everything before we right a single line of code as far as I'm concerned.

**Pull request merged**